src/style.css

```diff
@@ -0,0 +1,113 @@
+body {
+  --logo-big-size: 19px;
+  --logo-small-size: 11px;
+  --big--search-size: 70px;
+  --small--search-size: 42px;
+  --search-field-height: 19px;
+  --text-shadow: 0 4px 4px rgba(0,0,0,0.25);
+}
+
+.search--box {
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  height: var(--big--search-size);
+  display: flex;
+  box-sizing: border-box;
+  align-items: center;
+  margin-top: 20px;
+  padding-left: 26px;
+}
+
+.search--box:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
```

Comment:
are we really need 2 different hovers? just do only for box or bar

Reply:
I have only one `:hover` usage in my code, and it`s code, that u selected, no more